src/app/services/fiddle.service.spec.ts

```diff
@@ -1,11 +1,30 @@
 import {FiddleService} from './fiddle.service';
+import {Http, HTTP_PROVIDERS, Response} from 'angular2/http';
+import {  
+    expect, 
+    it, 
+    describe,
+    injectAsync,
+    TestComponentBuilder,
+    beforeEachProvider
```

Comment:
this is where we get the error that there is no exported module beforeEachProvider.

node_modules/angular2/testing shows this:
`export * from './src/testing/testing';`

Digging into that file shows this:
`export declare function beforeEachProviders(fn: any): void;`